packages/components/src/table/stories/table-summary-placeholder.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { Card, CardFooter } from '@wordpress/components';
```

Comment:
See the reason for using these two components here https://github.com/woocommerce/woocommerce-admin/pull/7294/files#r665976770

Reply:
Nice 👍